lib/ffmpeg/transcoderbase.py

```diff
@@ -210,6 +211,18 @@ def transcode(self):
           iopts.format(osencoded), oopts.format(vppscale, self.frames, osyuv))
         self.check_resolution(output, osencoded)
         self.check_metrics(yuv, refctx = [(n, channel)])
+
+        is_hdr = vars(self).get("HDR", 0)
+        if is_hdr:
+          output = call(
+            f"{exe2os('ffprobe')} -i {osencoded}"
+            f" -show_streams"
+          )
+          color_range_info = re.findall(r'(?<=color_range=)\w*', output)
+          color_space_info = re.findall(r'(?<=color_space=)\w*', output)
+          color_transfer_info = re.findall(r'(?<=color_transfer=)\w*', output)
+          color_primaries_info = re.findall(r'(?<=color_primaries=)\w*', output)
```

Comment:
Does this HDR meta change when it is transcoded?  If not, we should verify it matches the source HDR meta.